src/main/scala/org/broadinstitute/dsde/agora/server/AgoraConfig.scala

```diff
@@ -82,6 +82,9 @@ object AgoraConfig {
     lazy val licenseUrl = swagger.getString("licenseUrl")
     lazy val baseUrl = swagger.getString("baseUrl")
     lazy val apiDocs = swagger.getString("apiDocs")
+    lazy val clientId = swagger.as[Option[String]]("clientId").getOrElse("clientId")
+    lazy val realm = swagger.as[Option[String]]("realm").getOrElse(gcsProjectId)
+    lazy val appName = swagger.as[Option[String]]("appName").getOrElse(gcsProjectId)
```

Comment:
This could be even simpler since the same value is in `gcsProjectId`, but I'll test to see if the value in reference.conf is available to AgoraConfig.